internal/bundle/fmt.go

```diff
@@ -127,3 +132,31 @@ func FormatBundleData(data []byte) ([]byte, error) {
 
 	return Format(b)
 }
+
+func SortBundleData(data []byte, sortFields []string) ([]byte, error) {
```

Comment:
I would just call `SortBundle` and only accept bundle as input. I would sort by all parts automatically. The loading should not be done inside of the function. It would allow us to reuse the function way more often. So if we always sort by all in SortContentV2 and do not need the config setup. Wouldn't this make the whole implementation easier?